.circleci/config.yml

```diff
@@ -1193,9 +1192,6 @@ workflows:
     jobs:
       - backstop_reference:
           target: prod
-          filters:
-            branches:
-              only: develop
```

Comment:
Should we remove this?

Reply:
Yes. Anytime we send trigger_workflow we want this to run. And never otherwise.